.pre-commit-config.yaml

```diff
@@ -16,3 +16,7 @@ repos:
     hooks:
       - id: flake8
         args: ['--config=backend/setup.cfg']
+  - repo: https://github.com/pre-commit/pre-commit
+    rev: v2.7.1
+    hooks:
+      - id: validate_manifest
```

Comment:
why do we need this change? 😅

Reply:
I will try to commit without these changes, but I couldn't commit anything before I added these changes. some problems with pre-commit.

Reply:
maybe you have to install it again?